analytics/criteo.js

```diff
@@ -0,0 +1,109 @@
+//
+// Criteo tracking for auctions product feed and artworks product feed
+//
+
+window.criteo_q = window.criteo_q || []
+var pathSplit = location.pathname.split('/')
+var userEmail = function() {
+  return sd.CURRENT_USER ? [sd.CURRENT_USER.email] : [];
```

Comment:
Criteo requires we send the userEmail as the single element in an array, otherwise an empty array